pkg/auth/condition.go

```diff
@@ -72,6 +76,11 @@ func (op *IpAddressOperator) Evaluate(fields map[string][]string, conditionCtx *
 
 	// Check each field in the condition against context values (AND logic between fields)
 	for fieldName, conditionValues := range fields {
+		// If no values specified for this field, condition fails
+		if len(conditionValues) == 0 {
```

Comment:
Does hits mean if I set IpAddress == "" and action deny - it will always deny for example?